courses/machine_learning/deepdive2/structured/solutions/4a_sample_babyweight.ipynb

```diff
@@ -0,0 +1,2818 @@
+{
```

Comment:
<p>A few things to note here.</p><p> </p><p>1) These two select clauses are repeated later. If you set them as variables,  not only will it save space, but when there is a bug or optimization, you'll only have to update one place (the wday bug is still in hash_cols_fixed_query)</p><p> </p><p>2) If there are only two cases in a case statement, then COALESCE or IF is preferred, so go ahead and delete the case statement. If you think the students would learn better from a breakdown of what's happening, add a link to the documentation: <a href="https://cloud.google.com/bigquery/docs/reference/standard-sql/conditional_expressions" target="_blank">https://cloud.google.com/bigquery/docs/reference/standard-sql/conditional_expressions</a></p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/671/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done</p>